jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/DevToolsToolWindow.kt

```diff
@@ -154,8 +159,17 @@ class DevToolsToolWindow(private val project: Project) : SimpleToolWindowPanel(t
         state.applyTo(tree)
     }
 
-    fun makeServiceChildrenVisible() {
-        TreeUtil.expand(tree, 2)
+    fun showGotIt(node: String?, tooltip: GotItTooltip) {
+        TreeUtil.promiseExpand(tree, 2).onSuccess {
+            node ?: return@onSuccess
+            treeModel.invoker.invoke {
+                val path = treeModel.asSequence().firstOrNull { (it.userObject as? AbstractTreeNode<*>)?.value == node }?.path ?: return@invoke
+                runInEdt {
+                    val pointToRight = tree.getPathBounds(TreePath(path))?.let { Point(it.x + it.width, it.y + it.height / 2) } ?: return@runInEdt
```

Comment:
Would the width of the panel affect the got it tooltip?

Reply:
no the bounding box is determined by the length of the node text